extensions/authenticated/README.md

```diff
@@ -1,4 +1,4 @@
-# Authenticated
+# Authenticated extension
```

Comment:
It's redundant, in the marketplace you'll have `Authenticated extension`

Reply:
![CleanShot 2025-04-16 at 10 41 28@2x](https://github.com/user-attachments/assets/03cbbc14-720b-4056-9a4a-f7b35b7605fd)
that's what I want!

Reply:
So every library in npm/crates/etc.. should have `library`, `package`, `crate` suffix?

Reply:
well up to you

Reply:
You'll need to increase the version of each lib so that the readme gets pushed

Reply:
it's more about consistency and setting the standard for future extensions. I think it reads nicer than just the name of the extension which is already listed at the top of the page.

@tomhoule is going to bump all the extensions with the new SDK so leave it to him to update across the board.